sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -1063,8 +1063,7 @@ namespace Azure { namespace Storage { namespace Blobs {
           std::vector<uint8_t> bodyContent;
           if (bodyStream->Length() == -1)
```

Comment:
What does this mean?

Reply:
unknown length, it's Azure Core's design.